manifests/r/Roblox/RobloxStudio/1.0/Roblox.RobloxStudio.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Roblox.RobloxStudio
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Roblox | Published to database by Markandpreston
+PackageName: Roblox Studio
+License: Copyright © 2020 Roblox Corporation. All rights reserved.
+ShortDescription: 'Roblox Studio is not my product. Its for Roblox Corporation. | About Roblox Studio: https://en.wikipedia.org/wiki/Roblox#Roblox_Studio | Published to the database of winget by Markandpreston (all of my profiles in youtube, twitter... is Markandpreston.'

```

Comment:
```suggestion
ShortDescription: Roblox Studio lets you create anything and release with one click to smartphones, tablets, desktops, consoles, and virtual reality devices.
```

Reply:
You got it wrong. Roblox Studio is an app that lets you create games and publish them to Roblox. You didn't specificallysay that so it might confuse users.

Reply:
It's actually from the official website: https://www.roblox.com/create#:~:text=Roblox%20Studio%20lets%20you%20create%20anything%20and%20release%20with%20one%20click%20to%20smartphones%2C%20tablets%2C%20desktops%2C%20consoles%2C%20and%20virtual%20reality%20devices

If you want to provide feedback to the Roblox team to change it, you should create a post in https://devforum.roblox.com/.

Reply:
Take a look at Unity's package for ideas: https://github.com/microsoft/winget-pkgs/blob/7660956a0aa8701992a5fe7865b2a284ee63c2e9/manifests/u/UnityTechnologies/Unity/2021/2021.2.7f1/UnityTechnologies.Unity.2021.locale.en-US.yaml#L15

Reply:
thx will edit

Reply:
i fixed it #40073